### PR TITLE
Milestone 3: Portfolio Website

### PR DESCRIPTION
**Pull request by DannieMo**
This is the third milestone of my portfolio website. In this milestone, I added a contact form and footer with my coding partner, Rustamxon.😀

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -96,62 +139,49 @@ <h4>Multi-Purpose Stories</h4>
...
+                a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="contact" class="contact-form">
+            <div class="data">
+                <input type="text" class="contact-element" name="user-fname" placeholder="First Name"><br class="custom">
+                <input type="text" class="contact-element" name="user-lname" placeholder="Last Name">
+            </div>
+            <label for="email"></label>
+            <input type="email" id="email" class="contact-element1" name="user-email" placeholder="Email Address"><br>
+            <textarea name="user-message" class="contact-element2" cols="30" rows="10" placeholder="Write your message here"></textarea><br><br>
```
Comment (Shaher-11) [inline index.html:177]: Add some space between the mail field the message field 
#### Original design
right

#### Your Project
wrong

File: index.html [hunk 2]
```diff
@@ -96,62 +139,49 @@ <h4>Multi-Purpose Stories</h4>
...
+            <button type="button" class="contact-button">Seen</button>
```
Comment (Shaher-11) [inline index.html]: Change the button as shown below
#### Original design
right

#### Your Project
wrong

File: index.html [hunk 3]
```diff
@@ -96,62 +139,49 @@ <h4>Multi-Purpose Stories</h4>
...
+        </form>
+        <footer class="socials">
```
Comment (Shaher-11) [inline index.html:180]: Reduce the space between the from the social icons

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @DannieMo,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**